concourse-production.yml

```diff
@@ -28,3 +28,10 @@ instance_groups:
   azs: [z1]
   networks:
   - name: production-concourse
+- name: iaas-worker
+  vm_type: production-concourse-worker
+  vm_extensions: [production-concourse-iaas-profile]
+  instances: 1
```

Comment:
I'm conflicted over running one of anything in production vs having two machines that will be idle 99% of the time.